aiohttp/web_middlewares.py

```diff
@@ -23,6 +24,11 @@ def _check_request_resolves(request, path):
     return False, request
 
 
+def new_middleware(f):
```

Comment:
Hmm.
I don't like the name. We have `new_middleware` and have no `old_middleware`.
Mybe just `@middleware` is enough?

Reply:
I agree that I don't like `new_middleware`, I'll switch to `middleware`.

Reply:
+1